packages/localizations/src/de-DE.ts

```diff
@@ -554,3 +554,8 @@ export const deDe: LocalizationResource = {
     numeric: "{{ date | numeric('de-DE') }}",
   },
 } as const;
+
+
+export const deDE = deDe;
```

Comment:
@dimkl we also need to re-export `deDe` as `deDE`. Once we deprecate `deDe` we can fix the name properly.

Reply:
We could update the current export since it's internal and use the following in `packages/localizations/src/index.ts`:
```
/*
 * @deprecated use `deDE` instead
 */
export { deDE as deDe } from './de-DE';
export { deDE } from './de-DE';
```
With that we can limit the cleanup changes to be the deprecation warning and the `...deDE as deDe...` line.

Reply:
makes sense & good call. done